app/models/user.rb

```diff
@@ -5,23 +5,21 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  belongs_to :small_group, inverse_of: :users
-  has_many :assigned_routes, inverse_of: :user
-  has_many :comments, inverse_of: :user
-  has_many :maintenance_requests, inverse_of: :user
-  has_many :ratings, inverse_of: :user
-  has_many :routes, inverse_of: :user
-  has_many :support_tickets, inverse_of: :user
+  belongs_to :small_group
+  # has_many :assigned_routes
```

Comment:
The previous merge will need this to not be commented out so i can do things like:
```
@user = User.find(params[:id])
@assigned_routes = @user.assigned_routes.not_complete.not_hidden
```